taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service.settings import AUTH_USER_MODEL
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
Your value of max_length allocates an excessive amount of memory and I think this is not good, so you should reduce the value of max_length to 63 (I assume), because such fields don't require so many characters. You should fix it everywhere ;)